src/main/java/io/github/thebusybiscuit/slimefun4/core/services/BlockDataService.java

```diff
@@ -137,31 +137,31 @@ public boolean isTileEntity(@Nullable Material type) {
             return false;
         }
 
-        switch (type) {
-            case PLAYER_HEAD:
-            case PLAYER_WALL_HEAD:
-            case CHEST:
-            case DISPENSER:
-            case BREWING_STAND:
-            case DROPPER:
-            case FURNACE:
-            case BLAST_FURNACE:
-            case HOPPER:
-            case LECTERN:
-            case JUKEBOX:
-            case ENDER_CHEST:
-            case ENCHANTING_TABLE:
-            case DAYLIGHT_DETECTOR:
-            case SMOKER:
-            case BARREL:
-            case SPAWNER:
-            case BEACON:
+        return switch (type) {
+            case PLAYER_HEAD,
+                PLAYER_WALL_HEAD,
+                CHEST,
+                DISPENSER,
+                BREWING_STAND,
+                DROPPER,
+                FURNACE,
+                BLAST_FURNACE,
+                HOPPER,
+                LECTERN,
+                JUKEBOX,
+                ENDER_CHEST,
+                ENCHANTING_TABLE,
+                DAYLIGHT_DETECTOR,
+                SMOKER,
+                BARREL,
+                SPAWNER,
+                BEACON ->
                 // All of the above Materials are Tile Entities
-                return true;
-            default:
+                true;
+            default ->
                 // Otherwise we assume they're not Tile Entities
-                return false;
-        }
+                false;
+        };
```

Comment:
Again, would be nice if you could add a comment saying "TODO: Add Slimefun Tag here"